jsx/Form.js

```diff
@@ -1187,14 +1235,33 @@ class NumericElement extends Component {
     let disabled = this.props.disabled ? 'disabled' : null;
     let required = this.props.required ? 'required' : null;
     let requiredHTML = null;
+    let errorMessage = null;
+    let elementClass = 'row form-group';
+
+    // Add required asterix
+    if (required) {
+      requiredHTML = <span className="text-danger">*</span>;
+    }
```

Comment:
in **24.1**

_this seems to have been introduced in some capacity already in 24.1_